src/index.html

```diff
@@ -10,31 +10,31 @@
    <script src="https://kit.fontawesome.com/08b4cd9ea0.js" crossorigin="anonymous"></script>
 
    <title>Home Page</title>
-    <!-- <label for="check" class="hamberger-btn">
+   <!-- <label for="check" class="hamberger-btn">
             <a href="#"><i id="ham" class="fas fa-bars"></i></a>
          </label> -->
```

Comment:
Great job team🤝

- [ ] Kindly remove commented code from all your files as this is regarded as a bad practice,